### DISCUSSION TITLE
Meeting invitations

### DISCUSSION THREAD

**Discussion by zvyezdan** [nhn/tui.calendar#847] [Q&A]
Hello,

does TUI calendar support meeting invites out of the box? If so, does it support schedulable meeting invites where invitees can actually schedule the time.

It's a use case where I could schedule a series of meetings (e.g. interviews) over a selected period of time (e.g. 2 weeks) during business hours (e.g. 9am- 3pm), having each meeting go for 1 hour. Then I send the meeting invite to 50 users and they choose their time and when they do my calendar gets updated.

When booking the time users/invitees would have a special calendar view, of course, where they can proceed with scheduling.

I understand this may be a very special use case / scenario of a calendar/scheduler usage... but calendar services like calendly support this out of the box.

Thanks for your help!
Dan

**Comment by adhrinae** [comment]
@zvyezdan 
Hello, Dan.
Unfortunately, we only provide the calendar itself, not a whole schedule management service as you want.
At least [there is an `attendee` field in the `Schedule` model](https://nhn.github.io/tui.calendar/latest/Schedule), so you can leverage this to create your own management system.

So If you want to implement the meeting invitation feature with the `tui.calendar`, you might need to:

* Customize a schedule popup that can have the attendees field.
* Request to the server that can handle sending invitations and manage schedules. Usually, it happens [on `beforeCreateSchedule` event](https://nhn.github.io/tui.calendar/latest/Calendar#event-beforeCreateSchedule).
